public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module

```diff
@@ -73,10 +73,10 @@ function os2web_accessibility_frontpage($nid = '') {
     $print_node_id = $nid_setting;
   }
 
-  $print = print_controller($print_node_id, $query, $cid, PRINT_HTML_FORMAT);
-
+  $print =  print_controller($query, PRINT_HTML_FORMAT, $cid);
```

Comment:
`PRINT_HTML_FORMAT` is not defined anywhere, so it's NULL.
@juuliabellcom are you sure the functionality works as expected?

`$cid` is also not defined, but it's a nullable param. For clarify it's better to not include it at all here.

Reply:
@stanbellcom fixed https://github.com/syddjurs/syddjurs.dk/pull/169/commits/2670201384e129a508e11056f9de471e934b6e1d#diff-451abd137472af0eebe44aa8a5c4075cR76